prompts/configs.go

```diff
@@ -169,14 +187,24 @@ func configBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 	return &nextState, nil
 }
 
-func languageSpecificForms(language string, existingConfig *config.Configuration, configFields []config.SDKGenConfigField, isQuickstart bool) ([]huh.Field, []string, error) {
+func languageSpecificForms(language string, existingConfig *config.Configuration, configFields []config.SDKGenConfigField, isQuickstart bool, sdkClassName string) ([]huh.Field, []string, error) {
 	langConfig := config.LanguageConfig{}
 	if existingConfig != nil {
 		if conf, ok := existingConfig.Languages[language]; ok {
 			langConfig = conf
 		}
 	}
 
+	// Default the packageName to the SDKClassName
+	// TODO: This updates the placeholder but is intended to be a default value
+	// TODO: We should explain what this is by example ie:
```

Comment:
Again these descriptions are pulled from openapi-generation. Feel free to append to them if you want
https://github.com/speakeasy-api/speakeasy/blob/main/prompts/configs.go#L195

fields.description